openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -80,6 +80,7 @@ class CourseContext(TimeStampedModel):
     course_visibility = models.CharField(
         max_length=32, choices=[(constant.value, constant.value) for constant in CourseVisibility]
     )
+    days_early_for_beta = models.FloatField(null=True, blank=True)
```

Comment:
Ah, that's why... Can you please adjust this to be an integer as well, to match the change in `data.py`?

Reply:
Sorry about that! WIll fix asap.